### DISCUSSION TITLE
ModuleNotFoundError: No module named 'slugify'

### DISCUSSION THREAD

**Discussion by gmoschese** [cheshire-cat-ai/core#475] [Q&A] [answered]
Hi Guys 
during the installation procedure as official documentation, I received the follow error after docker compose up command:

cheshire_cat_vector_memory  | [2023-09-29T16:01:04.820Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
cheshire_cat_core           | Traceback (most recent call last):
cheshire_cat_core           |   File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
cheshire_cat_core           |     return _run_code(code, main_globals, None,
cheshire_cat_core           |   File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
cheshire_cat_core           |     exec(code, run_globals)
cheshire_cat_core           |   File "/app/cat/main.py", line 17, in <module>
cheshire_cat_core           |     from cat.looking_glass.cheshire_cat import CheshireCat
cheshire_cat_core           |   File "/app/cat/looking_glass/cheshire_cat.py", line 10, in <module>
cheshire_cat_core           |     from cat.mad_hatter.mad_hatter import MadHatter
cheshire_cat_core           |   File "/app/cat/mad_hatter/mad_hatter.py", line 12, in <module>
cheshire_cat_core           |     from cat.mad_hatter.plugin_extractor import PluginExtractor
cheshire_cat_core           |   File "/app/cat/mad_hatter/plugin_extractor.py", line 7, in <module>
cheshire_cat_core           |     from slugify import slugify
cheshire_cat_core           | ModuleNotFoundError: No module named 'slugify'
cheshire_cat_core exited with code 1_

Thank you
Giovanni

**Comment by nicola-corbellini** [comment, marked as answer]
have you tried updating the cat with `docker compose build --no-cache`?
